setup/action.yml

```diff
@@ -10,6 +10,8 @@ inputs:
   aws_secret_id:
     description: "The name of the aws secret to use"
     required: true
+  artifactory_username:
+    description: "The artifactory username to be used"
```

Comment:
What's this input used for? Do we need this in the final version?

Reply:
This was needed because the artifactory username was `go-driver`, which caused simple things like the draft github URL to be redacted.